src/erlfmt_format.erl

```diff
@@ -421,6 +519,8 @@ fa_groups([{op, Meta, '/', {atom, _, HeadFunction}, _} = Head0 | Tail]) ->
             [Head0 | fa_groups(Rest)];
         {Group, Rest} ->
             Head = erlfmt_scan:delete_annos([pre_comments, post_comments], Head0),
+            EndLoc = erlfmt_scan:get_end_location(lists:last(Group)),
+            Meta = erlfmt_scan:set_end_location(EndLoc, Meta0),
```

Comment:
In the parser we have a `?range_anno` macro that does something similar. What do you think about having a similar function in `erlfmt_scan`? Something like:
```erl
range_anno(First, Last) when is_tuple(First), is_tuple(Last) ->
    (element(2, First))#{end_location := get_end_location(Last)}.
```

We could call it here as:
```erl
Meta = erlfmt_scan:range_anno(Head, lists:last(Group))
```

Reply:
That sounds great :)

Reply:
We should probably also set the `inner_end_location`, right?

Reply:
Hmm... potentially. Do we really need it? It would need some `case`ing to only add it if it was there in `Last` in the first place.

Reply:
I added range_anno, but I needed to pass in Meta, otherwise, we lose the comments.